pkg/sdk/v1beta1/kafkacluster_types.go

```diff
@@ -353,6 +354,21 @@ type CommonListenerSpec struct {
 	ContainerPort int32  `json:"containerPort"`
 }
 
+// ListenerStatuses defines the internal and external listener addresses
+type ListenerStatuses struct {
+	InternalListeners ListenerStatusMap `json:"internalListeners,omitempty"`
+	ExternalListeners ListenerStatusMap `json:"externalListeners,omitempty"`
+}
+
+// ListenerStatus objects can be looked up in ListenerStatusMap by listener name
+type ListenerStatusMap map[string]ListenerStatus
```

Comment:
If there are no methods implemented for `ListenerStatusMap` than there is no need for introducing a new type.

```
type ListenerStatuses struct {
	InternalListeners map[string] ListenerStatus `json:"internalListeners,omitempty"`
	ExternalListeners map[string] ListenerStatus `json:"externalListeners,omitempty"`
}
```